{
  "Header": {
    "Original": "CVE-2021-30471 (A flaw was found in PoDoFo 0.9.7. An uncontrolled recursive call in Pd ...)",
    "ID": "CVE-2021-30471",
    "Description": "(A flaw was found in PoDoFo 0.9.7. An uncontrolled recursive call in Pd ...)"
  },
  "Annotations": [
    {
      "Original": "- libpodofo 0.9.8+dfsg-1 (bug #986793)",
      "Type": "package",
      "Package": "libpodofo",
      "Kind": "fixed",
      "Version": "0.9.8+dfsg-1",
      "BugNo": 986793
    },
    {
      "Original": "[bullseye] - libpodofo \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "bullseye",
      "Package": "libpodofo",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "[buster] - libpodofo \u003cno-dsa\u003e (Minor issue)",
      "Type": "package",
      "Release": "buster",
      "Package": "libpodofo",
      "Kind": "no-dsa",
      "Description": "Minor issue"
    },
    {
      "Original": "[stretch] - libpodofo \u003cpostponed\u003e (Minor issue; can be fixed in next update)",
      "Type": "package",
      "Release": "stretch",
      "Package": "libpodofo",
      "Kind": "postponed",
      "Description": "Minor issue; can be fixed in next update"
    },
    {
      "Original": "NOTE: https://sourceforge.net/p/podofo/tickets/131/",
      "Type": "NOTE",
      "Description": "https://sourceforge.net/p/podofo/tickets/131/"
    },
    {
      "Original": "NOTE: https://github.com/podofo/podofo/commit/14689c5b8a60d3450f154e66c92632947b73f619 (0.9.8)",
      "Type": "NOTE",
      "Description": "https://github.com/podofo/podofo/commit/14689c5b8a60d3450f154e66c92632947b73f619 (0.9.8)"
    }
  ]
}